python/paddle/distributed/fleet/meta_optimizers/sharding/utils.py

```diff
@@ -479,12 +514,18 @@ def is_opt_vars(var):
                 return True
         return False
 
+    def is_gradient_merge_vars(var):
```

Comment:
Have we recorded these kinds of hard coding or rules?

Reply:
yes.
this naive rule should be updated later.
the problem is that: grad@gradientmerge should be persistable in global scope but not to be saved. we need design  a method to distinguish  persistable-non-savable vars with persistable-savable vars